devel/to-f90/Portfile

```
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       compilers 1.0
+
+name            to-f90
+version         2000-03-21
+revision        0
+categories      devel
+platforms       darwin
+license         Noncommercial
```

According to the read me:

> This is PUBLIC DOMAIN code

So I would list the license as `public-domain`.
```suggestion
license         public-domain
```

---

Oops. My fault.